sample/host/src/AuthenticatedLayout.tsx

```diff
@@ -22,20 +25,35 @@ export default function AuthenticatedLayout() {
 
     const navigationItems = useNavigationItems();
 
-    const renderItem: RenderItemFunction = useCallback(({ content, linkProps, additionalProps: { highlight, ...additionalProps } }, index, level) => {
+    const renderLink: RenderLinkItemFunction = useCallback(({ label, linkProps, additionalProps: { highlight, ...additionalProps } }, index, level) => {
```

Comment:
renderLink renderMenu renderSection renderItem can be defined outside the component again